blocks/index.js

```diff
@@ -13,29 +8,6 @@ import './hooks';
 // Blocks are inferred from the HTML source of a post through a parsing mechanism
 // and then stored as objects in state, from which it is then rendered for editing.
 export * from './api';
-export * from './autocompleters';
-export * from './colors';
 
 export { default as editorMediaUpload } from './editor-media-upload';
-export { default as AlignmentToolbar } from './alignment-toolbar';
-export { default as Autocomplete } from './autocomplete';
-export { default as BlockAlignmentToolbar } from './block-alignment-toolbar';
-export { default as BlockControls } from './block-controls';
-export { default as BlockFormatControls } from './block-format-controls';
-export { default as BlockEdit } from './block-edit';
-export { default as BlockIcon } from './block-icon';
-export { default as ColorPalette } from './color-palette';
-export { default as ContrastChecker } from './contrast-checker';
-export { default as ImagePlaceholder } from './image-placeholder';
-export { default as InnerBlocks } from './inner-blocks';
-export { default as InspectorControls } from './inspector-controls';
-export { default as InspectorAdvancedControls } from './inspector-advanced-controls';
-export { default as PlainText } from './plain-text';
-export { default as MediaUpload } from './media-upload';
-export { default as RichText } from './rich-text';
-export { default as RichTextProvider } from './rich-text/provider';
-export { default as UrlInput } from './url-input';
-export { default as UrlInputButton } from './url-input/button';
 export { default as EditorSettings, withEditorSettings } from './editor-settings';
```

Comment:
Why do `editor-settings` and `editor-media-upload` remain?

Reply:
because I was planning to them separately. See #6500 for the editor settings. And both require the components to be moved first.